sceptre/environment.py

```diff
@@ -369,19 +372,18 @@ def _get_available_stacks(self):
         config_files = glob(os.path.join(
             self.sceptre_dir, "config", self.path, "*.yaml"
         ))
-        base_config_file_names = [
-            os.path.basename(config_file).split(".")[0]
-            for config_file in config_files
-        ]
         stack_basenames = [
-            basename for basename in base_config_file_names
-            if basename != "config"
+            os.path.splitext(os.path.basename(config_file))[0]
+            for config_file in config_files
         ]
-        available_stacks = [
-            "/".join([self.path, basename])
-            for basename in stack_basenames
+
+        if "config" in stack_basenames:
```

Comment:
`stack_basenames` doesn't word as a variable name if it includes a `config` item. Can we move this if statement to a filter in the list comp on 375? Or change the name